messaging/src/main/java/org/axonframework/deadline/SimpleDeadlineManager.java

```diff
@@ -121,38 +127,49 @@ public String schedule(@Nonnull Instant triggerDateTime,
                     TimeUnit.MILLISECONDS
             );
             scheduledTasks.put(deadlineId, scheduledFuture);
-        });
+        }));
 
         return deadlineMessageId;
     }
 
     @Override
     public void cancelSchedule(@Nonnull String deadlineName, @Nonnull String scheduleId) {
+        Span span = spanFactory.createInternalSpan(
```

Comment:
I'm not sure I'd expect the cancel operations to be wrapped in a span.
My reasoning behind this is that I expect it for message handling (command, event, query, deadline), message dispatching (command, event, query, deadline) or retrieval (events and snapshots).
 
Would you care to explain why you felt this to be valuable?

Reply:
With tracing you want to see what actions were taken in the system, depending on input. I think cancelling a deadline is one of those decisions, since it can also be done conditionally. It would be good to visualize this happening, and it's low effort.

Reply:
Makes total sense, thanks for that!